package.json

```diff
@@ -179,6 +176,26 @@
       "eventsource-polyfill"
     ]
   },
+  "jest": {
+    "collectCoverageFrom": [
```

Comment:
also add `"!build/**"` to be ignored by coverage

Could you not just include the app folder and not bother ignoring the other folders?

`"app/**/*.{js,jsx}"`